recipes/webhelpers/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "WebHelpers" %}
+{% set version = "1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: ea86f284e929366b77424ba9a89341f43ae8dee3cbeb8702f73bcf86058aa583
+
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py3k]
```

Comment:
Can this be `noarch: python` and the `- python <=2.7` below?

Reply:
Couldn't get it to build with `noarch: python and python <=2.7`. I got python package conflicts.

Reply:
It looks like markupsafe isn't on Python 2, so did you mean to skip Python 2 here, rather than skipping Python 3?

Reply:
Yeah, this seems like it should be noarch with python >=3.5

Reply:
Added noarch and a patch to resolve a python3.x/2.x issue with webhelpers.html involving an import of `urllib`